backend/src/database/migrations/1741166137543-chips-migration.ts

```diff
@@ -0,0 +1,26 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class ChipsMigration1741166137543 implements MigrationInterface {
+    name = 'ChipsMigration1741166137543'
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(`ALTER TABLE "personnel" ADD "chips_last_ping" date`);
+        await queryRunner.query(`ALTER TABLE "personnel" ADD "chips_last_action_date" date`);
+        await queryRunner.query(`ALTER TABLE "personnel" ADD "chips_profile_missing" boolean NOT NULL DEFAULT false`);
```

Comment:
Just double checking these be not null? What if someone has no chips profile? (you can disregard and mark as resolved if this is correct -just double checking)

Reply:
Not null is correct when profile missing defaults to false and will turn true
Not null for the others is fine I think, since I'm setting a default value. They're pretty safely ignored when there's no chips profile